{
	"id": "ALSA-2024:9192",
	"advisories": [
		{
			"content": {
				"id": "ALSA-2024:9192",
				"title": "Moderate: python3.11 security update",
				"description": "Python is an interpreted, interactive, object-oriented programming language, which includes modules, classes, exceptions, very high level dynamic data types and dynamic typing. Python supports interfaces to many system calls and libraries, as well as to various windowing systems.  \n\nSecurity Fix(es):  \n\n  * python: The zipfile module is vulnerable to zip-bombs leading to denial of service (CVE-2024-0450)\n  * python: cpython: Iterating over a malicious ZIP file may lead to Denial of Service (CVE-2024-8088)\n\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.  \n\nAdditional Changes:  \n\nFor detailed information on changes in this release, see the AlmaLinux Release Notes linked from the References section.\n",
				"severity": [
					{
						"type": "vendor",
						"source": "errata.almalinux.org",
						"vendor": "moderate"
					}
				],
				"references": [
					{
						"source": "errata.almalinux.org",
						"url": "https://access.redhat.com/errata/RHSA-2024:9192"
					},
					{
						"source": "errata.almalinux.org",
						"url": "https://access.redhat.com/security/cve/CVE-2024-0450"
					},
					{
						"source": "errata.almalinux.org",
						"url": "https://access.redhat.com/security/cve/CVE-2024-8088"
					},
					{
						"source": "errata.almalinux.org",
						"url": "https://bugzilla.redhat.com/2276525"
					},
					{
						"source": "errata.almalinux.org",
						"url": "https://bugzilla.redhat.com/2307370"
					},
					{
						"source": "errata.almalinux.org",
						"url": "https://errata.almalinux.org/9/ALSA-2024-9192.html"
					}
				],
				"published": "2024-11-12T00:00:00Z",
				"modified": "2024-11-18T18:11:47Z"
			},
			"ecosystems": [
				"alma:9"
			]
		}
	],
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2024-0450",
				"references": [
					{
						"source": "errata.almalinux.org",
						"url": "https://access.redhat.com/security/cve/CVE-2024-0450"
					}
				]
			},
			"ecosystems": [
				"alma:9"
			]
		},
		{
			"content": {
				"id": "CVE-2024-8088",
				"references": [
					{
						"source": "errata.almalinux.org",
						"url": "https://access.redhat.com/security/cve/CVE-2024-8088"
					}
				]
			},
			"ecosystems": [
				"alma:9"
			]
		}
	],
	"detection": [
		{
			"ecosystem": "alma:9",
			"criteria": {
				"operator": "OR",
				"criterions": [
					{
						"vulnerable": true,
						"fix_status": {
							"class": "fixed"
						},
						"package": {
							"name": "python3.11",
							"architectures": [
								"aarch64",
								"i686",
								"ppc64le",
								"s390x",
								"x86_64"
							]
						},
						"affected": {
							"type": "rpm",
							"range": [
								{
									"lt": "0:3.11.9-7.el9"
								}
							],
							"fixed": [
								"0:3.11.9-7.el9"
							]
						}
					},
					{
						"vulnerable": true,
						"fix_status": {
							"class": "fixed"
						},
						"package": {
							"name": "python3.11-debug",
							"architectures": [
								"aarch64",
								"i686",
								"ppc64le",
								"s390x",
								"x86_64"
							]
						},
						"affected": {
							"type": "rpm",
							"range": [
								{
									"lt": "0:3.11.9-7.el9"
								}
							],
							"fixed": [
								"0:3.11.9-7.el9"
							]
						}
					},
					{
						"vulnerable": true,
						"fix_status": {
							"class": "fixed"
						},
						"package": {
							"name": "python3.11-devel",
							"architectures": [
								"aarch64",
								"i686",
								"ppc64le",
								"s390x",
								"x86_64"
							]
						},
						"affected": {
							"type": "rpm",
							"range": [
								{
									"lt": "0:3.11.9-7.el9"
								}
							],
							"fixed": [
								"0:3.11.9-7.el9"
							]
						}
					},
					{
						"vulnerable": true,
						"fix_status": {
							"class": "fixed"
						},
						"package": {
							"name": "python3.11-idle",
							"architectures": [
								"aarch64",
								"i686",
								"ppc64le",
								"s390x",
								"x86_64"
							]
						},
						"affected": {
							"type": "rpm",
							"range": [
								{
									"lt": "0:3.11.9-7.el9"
								}
							],
							"fixed": [
								"0:3.11.9-7.el9"
							]
						}
					},
					{
						"vulnerable": true,
						"fix_status": {
							"class": "fixed"
						},
						"package": {
							"name": "python3.11-libs",
							"architectures": [
								"aarch64",
								"i686",
								"ppc64le",
								"s390x",
								"x86_64"
							]
						},
						"affected": {
							"type": "rpm",
							"range": [
								{
									"lt": "0:3.11.9-7.el9"
								}
							],
							"fixed": [
								"0:3.11.9-7.el9"
							]
						}
					},
					{
						"vulnerable": true,
						"fix_status": {
							"class": "fixed"
						},
						"package": {
							"name": "python3.11-test",
							"architectures": [
								"aarch64",
								"i686",
								"ppc64le",
								"s390x",
								"x86_64"
							]
						},
						"affected": {
							"type": "rpm",
							"range": [
								{
									"lt": "0:3.11.9-7.el9"
								}
							],
							"fixed": [
								"0:3.11.9-7.el9"
							]
						}
					},
					{
						"vulnerable": true,
						"fix_status": {
							"class": "fixed"
						},
						"package": {
							"name": "python3.11-tkinter",
							"architectures": [
								"aarch64",
								"i686",
								"ppc64le",
								"s390x",
								"x86_64"
							]
						},
						"affected": {
							"type": "rpm",
							"range": [
								{
									"lt": "0:3.11.9-7.el9"
								}
							],
							"fixed": [
								"0:3.11.9-7.el9"
							]
						}
					}
				]
			}
		}
	],
	"data_source": {
		"id": "alma-errata",
		"raws": [
			"vuls-data-raw-alma-errata/9/2024/ALSA-2024:9192.json"
		]
	}
}
